packages/ibm-products/src/components/DecoratorBase/DecoratorBase.mdx

```diff
@@ -1,11 +1,7 @@
-{/* cspell:ignore setlabeltitle */}
-import { Story, ArgsTable, Source, Canvas } from '@storybook/addon-docs';
-import {
-  getStoryId,
-  CodesandboxLink,
-} from '../../global/js/utils/story-helper';
-import { DecoratorBase } from '.';
-import * as DecoratorBaseStories from './DecoratorBase.stories';
+{/* cspell:ignore setlabeltitle */} import { Story, ArgsTable, Source, Canvas }
+from '@storybook/addon-docs'; import { getStoryId, CodesandboxLink, } from
+'../../global/js/utils/story-helper'; import { DecoratorBase } from '.';
+import \* as DecoratorBaseStories from './DecoratorBase.stories';
```

Comment:
the formatting here is all bunched up and makes it harder to read the imports, is this intentional?

Reply:
definitely not whoops